alma/controllers/admin/AdminAlmaRefunds.php

```diff
@@ -99,7 +96,7 @@ public function ajaxProcessRefund()
         $totalOrder = null;
         $totalOrderAmount = null;
         try {
-            $refundResult = $this->runRefund($paymentId, $amount, $isTotal);
+            $refundResult = $this->runRefund($paymentId, $amount, false);
```

Comment:
it si pretty curious that `runRefund` is never used with `isTotal` context...
we will discuss about that to understand why before finalise the PR 😉